ngraph/core/include/ngraph/op/gru_sequence.hpp

```diff
@@ -0,0 +1,75 @@
+//*****************************************************************************
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "ngraph/op/op.hpp"
+#include "ngraph/op/util/rnn_cell_base.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v4
+        {
+            class NGRAPH_API GRUSequence : public Op, public util::RNNCellBase
```

Comment:
May be we should change the inheritance scheme and make `util::RNNCellBase` to be inherited from `Op` class?

@ilyachur , @GlebKazantaev , what do you think about this? Will it break backward compatibility?

Reply:
Yes, I guess we should change the inheritance.

Reply:
inputs are not passed to `util::RNNCellBase`, classes inherited from `util::RNNCellBase` can have different inputs, the class is only used to store common attributes.

Reply:
done in the separate PR: https://github.com/openvinotoolkit/openvino/pull/2000